examples/usecase-passport/keystone.ts

```diff
@@ -0,0 +1,25 @@
+import "dotenv/config";
+import { config } from "@keystone-6/core";
+import { lists } from "./schema";
+import { withAuth, session, createAuthenticationMiddleware } from "./auth";
+import { fixPrismaPath } from "../example-utils";
+
+export default withAuth(
```

Comment:
Any reason for wrapping `withAuth` here instead of just adding the config directly - seeing as it is an example, the `withAuth` abstraction feels confusing?